telco-hub/configuration/reference-crs/required/acm/kustomization.yaml

```diff
@@ -17,7 +16,6 @@ resources:
   - thanosSecretPolicy.yaml
   - thanosSecretPlacement.yaml
   - thanosSecretPlacementBinding.yaml
-  - thanosSecretMCSB.yaml
   # - observabilitySecret.yaml
```

Comment:
Is this intended to be commented out also?

Reply:
That secret is actually put in place by https://github.com/openshift-kni/telco-reference/blob/main/telco-hub/configuration/reference-crs/required/acm/pullSecretPolicy.yaml 
So commenting this out is correct. The underlying CR should be removed in preference of the policy though. Follow-on PR...